### DISCUSSION TITLE
Where is the image of the past elementary OS?

### DISCUSSION THREAD

**Discussion by aXdey65i** [elementary/triage#603] [Q&A]
The other day, I was the one who started supporting the beginner’s OS.

As a result of trying Elementary OS on Raspberry Pi, there was a problem and I couldn’t continue, so I want to try again with the image of Elementary OS 6.0 or 6.1, but Elementary O for Raspberry Pi Please give me the image archive of S6.0 and 6.1!!

Thank you very much!!

**Comment by ryonakano** [comment]
We don't provide previous release images officially, but there are community-made image of elementary OS 5.1 Hera for Raspberry Pi 4 available here:

https://github.com/meisenzahl/elementary-os-unofficial
